src/gc/gccommon.cpp

```diff
@@ -62,7 +62,7 @@ uint8_t g_build_variant = 1;
 uint8_t g_build_variant = 2;
 #endif // defined(BUILDENV_DEBUG)
 
-VOLATILE(int32_t) m_GCLock = -1;
+VOLATILE(int32_t) g_no_gc_lock = -1;
```

Comment:
>g_no_gc_lock [](start = 18, length = 12)

this shouldn't be in gccommon.cpp - it's only used internally by GC so should be in gc.cpp.